docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,405 @@
+| service_account    | n/a         | `string` | n/a                 |   yes    |
+| uid_operator_image | n/a         | `string` | n/a                 |   yes    |
+| uid_api_token      | n/a         | `string` | n/a                 |   yes    |
+| region             | n/a         | `string` | `"asia-southeast1"` |    no    |
+| network_name       | n/a         | `string` | `"uid-operator"`    |    no    |
+| uid_machine_type   | n/a         | `string` | `"n2d-standard-16"` |    no    |
+| uid_deployment_env | n/a         | `string` | `"integ"`           |    no    |
+| max_replicas       | n/a         | `number` | `5`                 |    no    |
+| min_replicas       | n/a         | `number` | `1`                 |    no    |
+| debug_mode         | n/a         | `bool`   | `false`             |    no    |
+
+#### Outputs
+
+| Name             | Description |
+|------------------|-------------|
+| load-balancer-ip | n/a         |
+
+#### Terraform Template&#8212;Notes
+
+1. You might want to change the load balancer from HTTP to HTTPS. To do this, follow these steps: (**GWH_YS19 surely they would have to use HTTPS? Should this be higher profile than a note at the end?**)
+
+   - Provide your cert via terraform following this page:
+  [google_compute_ssl_certificate](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_ssl_certificate.html)
+
+   - Then add the following configuration values in `module "gce-lb-http"` (**GWH_YS20 is the syntax correct here? Is it a name/value pair? Don't understand this. Module not previously mentioned. It's a module of what?**)
+
+     ```
+       ssl                  = true
+       ssl_certificates     = [google_compute_ssl_certificate.you_cert.self_link]
+       use_ssl_certificates = true
+       https_redirect       = true
+     ```
+
+2. Follow the [Prerequisites](https://github.com/IABTechLab/uid2-operator/tree/master/scripts/gcp-oidc#prerequisites)
+   section to set up a service account. (**GWH_YS21 this seems a little unkind to make the last step something they still need to do. Is this really all the information needed? Will it all make sense to our users and be easy to follow and implement?**)
+
+## Upgrade
+
+When a new version of UID2 Google Cloud Platform Confidential Space is released, private operators receive an email notification of the update, with a new image version. There is a window of time for upgrade, after which the older version is deactivated and is no longer supported.
+
+If you're upgrading to a new version, all you need to do is update the `{IMAGE_SHA}` in your configuration ([Integration Deployment](#integration-deployment) and/or [Production Deployment](#production-deployment)) to the new value from the update notification.
+
+(**GWH_YS22 Not sure that the addition of the Terraform template info resolved Andrei's comment re upgrade and how it would be done -- and we'd have to cover with and without load balancing if it's only an option.**)
```

Comment:
If they go with Terraform way, they only need to update the Image sha in input and all are set.

If they go with GCloud command way, they need to manually bring up instances and shutdown old instances ( also manage the LB mapping if they have. ) 

That's why we recommend to deploy prod in terraform way.

@atarassov-ttd not sure how existing GCP private operator partners manage their deployment since we only provide Gcloud command to bring up single instance in old doc.